examples/pytorch/graphsaint/train_sampling.py

```diff
@@ -0,0 +1,204 @@
+        model, g, labels, test_mask, multilabel)
+    print("Test F1-mic {:.4f}, Test F1-mac {:.4f}".format(test_f1_mic, test_f1_mac))
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='GCN')
+    # data source params
+    parser.add_argument("--dataset", type=str, choices=['ppi', 'flickr', 'reddit', 'yelp', 'amazon'], default='ppi',
+                        help="Name of dataset.")
+
+    # cuda params
+    parser.add_argument("--gpu", type=int, default=-1,
+                        help="GPU index. Default: -1, using CPU.")
+
+    # sampler params
+    parser.add_argument("--sampler", type=str, default="node", choices=['node', 'edge', 'rw'],
+                        help="Type of sampler")
+    parser.add_argument("--node-budget", type=int, default=6000,
+                        help="Expected number of sampled nodes when using node sampler")
+    parser.add_argument("--edge-budget", type=int, default=4000,
+                        help="Expected number of sampled edges when using edge sampler")
+    parser.add_argument("--num-roots", type=int, default=3000,
+                        help="Expected number of sampled root nodes when using random walk sampler")
+    parser.add_argument("--length", type=int, default=2,
+                        help="The length of random walk when using random walk sampler")
+    parser.add_argument("--num-repeat", type=int, default=50,
+                        help="Number of repeating sampling one node to estimate edge / node probability")
+
+    # model params
+    parser.add_argument("--n-hidden", type=int, default=512,
+                        help="Number of hidden gcn units")
+    parser.add_argument("--arch", type=str, default="1-0-1-0",
+                        help="Network architecture. 1 means an order-1 layer (self feature plus 1-hop neighbor "
+                             "feature), and 0 means an order-0 layer (self feature only)")
+    parser.add_argument("--dropout", type=float, default=0,
+                        help="Dropout rate")
+    parser.add_argument("--batch-norm", action='store_true',
+                        help="Whether to use batch norm")
+    parser.add_argument("--aggr", type=str, default="concat", choices=['mean', 'concat'],
+                        help="How to aggregate the self feature and neighbor features")
+
+    # training params
+    parser.add_argument("--n-epochs", type=int, default=100,
+                        help="Number of training epochs")
+    parser.add_argument("--lr", type=float, default=0.01,
+                        help="Learning rate")
+    parser.add_argument("--weight-decay", type=float, default=0,
+                        help="Weight for L2 reg")
+    parser.add_argument("--val-every", type=int, default=1,
+                        help="Frequency of evaluation on the validation set in number of epochs")
+    parser.add_argument("--note", type=str, default='none',
+                        help="Note for log dir")
```

Comment:
This argument can be confusing. Either you can change this to `--log-dir` or replace the help note with "the log results will be saved in ./log/args.dataset/args.note"